TombEngine/Objects/TR4/Entity/tr4_demigod.cpp

```diff
@@ -709,6 +730,15 @@ namespace TEN::Entities::TR4
 					TriggerShockwave((Pose*)&pos, 24, 88, 256, 128, 128, 128, 32, EulerAngles::Zero, 8, true, false, (int)ShockwaveStyle::Normal);
 					TriggerHammerSmoke(pos.x, pos.y + 128, pos.z, 8);
 
+					Demigod1Data.LightPosition = Vector3i(pos.x, pos.y, pos.z);
```

Comment:
As I said above, you should not use a global.

Reply:
fixed